alphagov/paas-cf
# [#113608503] Remove our own simple_app_test

## What

This was added when we introduced the custom acceptance test suite alongside
the existing Cloud Foundry acceptance tests. It duplicates functionality in
the CF tests and was a placeholder until we added other tests to our custom
suite.

e18e80b adds our first real custom acceptance test so we no longer need to
have this. We need to keep `tests/acceptance-tests/example_apps` though
because it's used by that new HTTP test.

## How to review

1. Checkout this branch:

    ```
113608503-remove_test_acceptance_test
```
1. Deploy the pipelines from branch:

    ```
BRANCH=$(git rev-parse --abbrev-ref HEAD) make dev pipelines
```
1. Run the `create-bosh-cloudfoundry` pipeline and confirm that the `custom-acceptance-tests` job passes.

## Who can review

Anyone but @dcarley
